packages/grub/bios.cfg

```
@@ -5,6 +5,7 @@ gptprio.next -d boot_dev -u boot_uuid
 set root=$boot_dev
 set prefix=($root)/grub
 export boot_uuid
+search --no-floppy --set private --part-label BOTTLEROCKET-PRIVATE
```

Why `--part-label` works for grub 2.06? It isn't documented [here](https://www.gnu.org/software/grub/manual/grub/html_node/search.html), and I couldn't find references to it in the source code:

```fish
grub-2.06 via 🐍 v3.10.1 ❯ rg "\-\-part-label" -q; or echo "NOT FOUND"
NOT FOUND
grub-2.06 via 🐍 v3.10.1 ❯ rg "\-\-label" -q; and echo "FOUND"
FOUND
```

The docs mention you could use:

```suggestion
search.fs_label BOTTLEROCKET-PRIVATE --no-floppy --set private
```

---

Bottlerocket carries a [few](https://github.com/bottlerocket-os/bottlerocket/blob/develop/packages/grub/0014-gpt-add-search-by-partition-label-and-uuid-commands.patch) [patches](https://github.com/bottlerocket-os/bottlerocket/blob/develop/packages/grub/0017-gpt-add-search-by-disk-uuid-command.patch) that add this functionality.

`BOTTLEROCKET-PRIVATE` is a partition label, so searching by filesystem label won't find it.